app/models/planning_application.rb

```diff
@@ -279,13 +279,31 @@ def result_present?
   end
 
   def validation_requests
-    (description_change_validation_requests + replacement_document_validation_requests + additional_document_validation_requests + other_change_validation_requests + red_line_boundary_change_validation_requests).sort_by(&:created_at).reverse
+    (replacement_document_validation_requests + additional_document_validation_requests + other_change_validation_requests + red_line_boundary_change_validation_requests).sort_by(&:created_at).reverse
   end
 
   def cancelled_validation_requests
     validation_requests.filter(&:cancelled?).sort_by(&:cancelled_at).reverse
   end
 
+  def open_description_change_requests
+    description_change_validation_requests.open
+  end
+
+  def responded_description_request
+    description_change_validation_requests.closed.select do |request|
+      request.rejection_reason != "Request cancelled by planning officer." && request.auto_closed.blank?
+    end.last
+  end
+
+  def auto_closed_description_requests
+    description_change_validation_requests.select(&:auto_closed?).last
+  end
```

Comment:
method name is misleading in that it expects multiple requests where here you are just pulling out the last record?
`latest_auto_closed_description_request` ?